gsl_core/models.py

```diff
@@ -181,6 +181,19 @@ def clean(self):
         if errors:
             raise ValidationError(errors)
 
+    def contains(self, other_perimetre):
```

Comment:
Mon neurone matheux est légèrement déçu que l'opérateur [`⊂`](https://en.wikipedia.org/wiki/Subset) n'existe pas en Python, mais je m'en remettrai.

Reply:
Bah si voyons 😄 `x in y` appelle `y.__contains__(x)`: https://docs.python.org/3/reference/expressions.html#in

Reply:
OH

Reply:
Évidemment.